public/css/style.css

```diff
@@ -164,11 +170,33 @@ a {
   transform: translateY(-0.3rem);
   transition: all 0.3s;
 }
+/* .btn{
+  padding: 10px 25px;
+  font-size: 16;
+  font-weight: 500;
+  text-decoration: none;
+  color: avr();
+  background-color: transparent;
+  line-height: 1.5;
+  cursor: pointer;
+  border-radius: 30px;
+  color: #7158e2;
+  /* border: 2px solid red; */
+ /* transition: all 0.5s ease;;
+  display: inline-block
+}*/
+.btn:after{
+  border-radius: 30px;
+  /* color: var(--skin-color)  */
+ } 
```

Comment:
unused css code

Reply:
It is for the effects after hovering the button